nailgun/entities.py

```diff
@@ -4378,44 +4378,6 @@ def errata_applicability(self, synchronous=True, timeout=None, **kwargs):
         response = client.put(self.path('errata/applicability'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
 
-    def errata_apply(self, synchronous=True, timeout=None, **kwargs):
```

Comment:
Can we remove errata_applicability method too as it is also removed right?

Reply:
Negative. Errata applicability recalculation functionality remains present.
```
[GET /api/hosts/:host_id/errata]    List errata available for the content host
[GET /api/hosts/:host_id/errata/:id]    Retrieve a single errata for a host
[PUT /api/hosts/:host_id/errata/applicability]    Force regenerate applicability.
```